internal/db/storage/blob/s3/writer/option.go

```diff
@@ -47,28 +51,38 @@ func WithService(s *s3.S3) Option {
 	}
 }
 
+// WithBucket returns the option to set bucket for writer.
 func WithBucket(bucket string) Option {
 	return func(w *writer) {
-		w.bucket = bucket
+		if len(bucket) != 0 {
+			w.bucket = bucket
+		}
 	}
 }
 
+// WithKey returns the option to set key for writer.
 func WithKey(key string) Option {
 	return func(w *writer) {
-		w.key = key
+		if len(key) != 0 {
+			w.key = key
+		}
 	}
 }
 
+// WithContentType returns the option to set ct for writer.
 func WithContentType(ct string) Option {
 	return func(w *writer) {
-		if ct != "" {
+		if len(ct) != 0 {
 			w.contentType = ct
 		}
 	}
 }
 
+// WithMaxPartSize returns the option to set max for writer.
 func WithMaxPartSize(max int64) Option {
 	return func(w *writer) {
-		w.maxPartSize = max
+		if max > s3manager.MinUploadPartSize {
+			w.maxPartSize = max
```

Comment:
should we return error here or just ignore it?

Reply:
or only log it?

Reply:
I think we should return the error. because Vlad has a lot of value of config.
so, Therefore, I think that it will not start because an error occurs just by setting values ​​different. Therefore, since there is a default value, I think that the current implementation is OK.

Reply:
I think it is dangerous to use the default value without user attention.
@vankichi what do you think? should we ignore it, log it, or return error in this case?